web/docs/formats/json.md

```diff
@@ -233,6 +234,39 @@ With `--omit-empty-lists`, this example becomes:
 
 This options combines all other `--omit-*` options.
 
+### `--arrays-of-objects` (Printer)
+
+Prints one array of objects per batch of events arriving at the printer as
+opposed to printing one object per event.
```

Comment:
You should probably split the content here: "For the parser, this option does X. For the printer, it does Y."

Reply:
We discussed this quickly at the office and prefer the separate entries because they follow the order of the synopsis.